### ISSUE TITLE
Error during pre-loading phase

### ISSUE THREAD

**Issue by Coraline774477** [Traben-0/Entity_Texture_Features#166]
Minecraft 1.19.2
Forge: 43.2.8
ETF: 4.4.0

Log: https://gist.github.com/Coraline774477/b1a0105fda5932ca003935b8e800494a
Crash Report: N/A 
(even though the game did crash but it happened during the pre-loading phase which possibly means that the game couldn't record the crash)

**Comment by Coraline774477** [comment]
I'll try launching the game again but if the same thing happens then I have two options;

1) Downgrade ETF
2) Remove ETF for the time being till this issue is resolved

**Comment by Traben-0** [comment]
this is an emf issue it seems, and should be fixed with the latest emf version 0.2.8

**Comment by Coraline774477** [comment]
Oh, thought that this was caused by ETF since I had accidentally installed the 1.19.4 version of ETF 4.4.0 since I apparently didn't scroll down to look for the actual corporate that initially caused the crash
If it was what I thought it was the game would've told me that I had the 1.19.4 version of ETF installed instead of the 1.19.2 version, since I'm currently playing 1.16.5 and 1.19.2

Btw when 1.20 releases I'll be dropping 1.16.5 permanently, sure I can continue to play 1.16.5 but I've played on that version for too long now and I think it's time to put it to rest

But even when 1.20 releases I wont be playing modded 1.20 just like I'm not playing 1.19.4 since I'm missing 25 mods that haven't been ported to the latest version of minecraft yet, but I'm not waiting for Modern UI to port to 1.19.4 anymore since the author of the mod will have to rework their rendering since apparently Mojang changed the rendering engine again

Maybe that's why the last 25 mods still haven't gotten ported yet, due to the rendering engine being changed again which takes time to adjust to the new engine, especially mods like Modern UI

Ik that both mods didn't get started in 1.16.5 but I'm just sayin that I'll be abandoning that version permanently so that I could be with everyone else as well as some mods just like yours had started in 1.18 like both Player Companions and Human Companions
The player companions would've made my 1.16.5 world easier to deal with with the Apotheotic bosses from Apotheosis
Even tho those guys would absolutely annihilate my samurais cause some of those apotheotic bosses are truly powerful!
Like if their hp is at or above 200 they'll deal heavy damage or potentially one shot you even while wearing diamond armor.
One time while I was fighting with a zombified piglin in the overworld (a apotheotic boss btw) he hit me and literally launched me high into the air!

So a word of advice if you ever try out Apotheosis 1.16.5 or even 1.19.2 do NOT go toe to toe with one of these guys if you're not ready for a fight to the literal death (your death), when fighting with one of these guys make sure that your ontop of something that they cant reach you and cant pathfind to you or else you'll have a bad time
Believe me cause one time on a different 1.16.5 world befoew 1.16.5 was probably at its peak in popularity that I was fighting against a apotheotic boss skeleton or stray (maybe was a apotheotic boss stray) and I died several times tryin to kill him and I was wearing a full suit of non-enchanted iron armor while using weapons that weren't capable of killing him and he probably had over 200 hp since he kicked my teeth in three times before my fourth attempt was successful...
